.circleci/config.yml

```diff
@@ -70,10 +75,25 @@ jobs:
             docker load -i ./docker-images/spring-cloud-kubernetes-discoveryserver.tar
             docker load -i ./docker-images/spring-cloud-kubernetes-configserver.tar
       - run:
-          name: Run Kind Integration Tests
+          name: testcontainers integration tests
           command: |
+            # we need sort, because the last project calls K3S::stop
```

Comment:
Can you elaborate on this?  Why does the last project call `stop`?

Reply:
we don't, you are correct. I removed this part